src/index-backend.ts

```diff
@@ -46,7 +46,7 @@ if (process.argv[0].match(/\/ts-node$/)) {
   };
   Promise.resolve(test)
     .then(assertIs(FrontendResponseModel))
-    .then(res => prepareResponseForStorage(res, 'FI'))
+    .then(res => prepareResponseForStorage(res, 'FI', Promise.resolve('fake-secret-pepper')))
```

Comment:
I don't understand what this is and why this is needed here, would it maybe need a comment to explain?

Reply:
You're right, this isn't really needed here anymore.

That code path was used to test the validation rules of the backend by running the backend code locally via `ts-node`, as we didn't (and still don't) have a simple way of spinning up the backend locally, without deploying to Lambda.

Now that that we have the test suite AND a lint task that checks TS compilation, this isn't really needed.

Will try to remember to get rid of it. :wink: